src/objects/strategy-cards/strategy-card.js

```diff
@@ -27,16 +27,20 @@ function getTopLevelWidget(element) {
  * their removal in case of the destruction of the object.
  * The play card is mandatory while the
  *
- * @param refObject The object instance of the strategy card
- * @param widgetFactory A factory creating the UI for each player
- * @param [onStrategyCardPlayed] Event handler of the "Play" button pressed.
+ * @param {GameObject} refObject The object instance of the strategy card
+ * @param {function} widgetFactory A factory creating the UI for each player
+ * @param {Integer} height Height of the UI for each player
+ * @param {Color} color Color of the UI for each player
+ * @param {function} [onStrategyCardPlayed] Event handler of the "Play" button pressed.
  * This function only is required if the Play button should do more than just creating a UI.
  * The whole UI creation should belong into the widgetFactory to separate concerns.
- * @param [onStrategyCardSelectionDone] Event handler for the close
+ * @param {function} [onStrategyCardSelectionDone] Event handler for the closing event.
  */
 function registerStrategyCard(
```

Comment:
this is ok right now, it *might* be worth thinking about a builder style so you can easily set just the fields you want instead of having different styles of registration functions.  For instance:
```
class RegisterStrategyCard {
    setStrategyCardObject(obj) { ...; return this }
    setWidgetFactory(factory) { ...; return this }
    ...
    register() {...}
}
```
then when registering:
```
new RegisterStrategyCard()
    .setStrategyCardObject(refObject)
    .setWidgetFactory(factory)
    ...
    .register()
```

I'm no suggesting you redo it, but that style is nicely self-documenting and less headache for adding new arguments later.

Reply:
wish granted in #71